backend/src/ee/services/audit-log-stream/audit-log-stream-service.ts

```diff
@@ -62,9 +60,7 @@ export const auditLogStreamServiceFactory = ({
     );
     ForbiddenError.from(permission).throwUnlessCan(OrgPermissionActions.Create, OrgPermissionSubjects.Settings);
 
-    if (appCfg.isCloud) {
-      blockLocalAndPrivateIpAddresses(url);
-    }
+    await blockLocalAndPrivateIpAddresses(url);
```

Comment:
wait why did we remove the conditional here? we want to allow localhost for audit log streams of self-hosted right? like if their network is internal and they have like an ingester or something